apps/ledger-live-mobile/src/analytics/TrackScreen.tsx

```diff
@@ -21,8 +27,14 @@ export default function TrackScreen({ category, name, ...props }: Props) {
     if (isFocusedRef.current !== isFocused) {
       isFocusedRef.current = isFocused;
       if (isFocusedRef.current) {
-        previousRouteNameRef.current = currentRouteNameRef.current;
-        currentRouteNameRef.current = `${category + (name ? ` ${name}` : "")}`;
+        if (refreshSource) {
+          previousRouteNameRef.current = currentRouteNameRef.current;
+          currentRouteNameRef.current = `${
+            category + (name ? ` ${name}` : "")
```

Comment:
You should add the same for `category`?

Reply:
IDK, because TrackScreen will have a bad result if no category is provided...